samples/bluetooth/central_and_peripheral_hr/README.rst

```diff
@@ -76,7 +74,7 @@ Building and running
 Testing
 =======
 
-After programming the sample to your development kit, you can test it either by connecting to other development kits that are running the :ref:`peripheral_hids_keyboard` sample, or by using the `nRF Connect for Desktop`_ **Bluetooth Low Energy** application that emulates a HRS server.
+After programming the sample to your development kit, you can test it either by connecting to other development kits that are running the :ref:`zephyr:peripheral_hr` sample, or by using the `nRF Connect for Desktop`_ **Bluetooth Low Energy** application that emulates a HRS server.
```

Comment:
```suggestion
After programming the sample to your development kit, test it either by connecting to other development kits that are running the :ref:`zephyr:peripheral_hr` sample, or by using the `nRF Connect for Desktop`_ **Bluetooth Low Energy** application that emulates a HRS server.
```

Reply:
Fixed. Also change "a" to "an" before "HRS" - hope that's ok.